### ISSUE TITLE
[Radio] Input radio not focusable with keyboard

### ISSUE THREAD

**Issue by MartinWeb** [AxaFrance/react-toolkit#1039]
### Issue and Steps to Reproduce

Using the tab key doesn't focus the different radio inputs of a group.

#### Versions

2.0.0

#### Expected

Inputs should be focus for accessibility purposes.
